api/serverless.ts

```diff
@@ -111,7 +111,7 @@ const serverlessConfiguration: AWS = {
   ],
   provider: {
     name: "aws",
-    runtime: "nodejs16.x",
+    runtime: "nodejs18.x",
```

Comment:
We've updated other parts of our app to run Node `20.10.0`. It looks like there's an [AWS Lambda runtime for nodejs20.x](https://aws.amazon.com/blogs/compute/node-js-20-x-runtime-now-available-in-aws-lambda/). Is there a reason you picked 18?

I'm not opposed. Just curious.

Reply:
I knew about this and tried it, but our version of `serverless` doesn't support it, and upgrading all our `serverless`-related dependencies was well out of scope of this ticket.

I'll add this to the release process doc though, and whoever gets a `serverless`-related Renovate should try making `nodejs20.x` work.